scripts/respec.sh

```diff
@@ -17,11 +17,11 @@ RESPEC_SOURCE="$SPEC_PATH/specification-respec.html"
 RESPEC_STATIC="$SPEC_PATH/index.html"
 
 # "$PRETTIER_PATH" "$SPEC_PATH/includes" --write
-"$TIDY_PATH" -config "$SPEC_PATH/tidyconfig.txt" -m "$RESPEC_SOURCE"
+"$TIDY_PATH" -config "$REPO_ROOT/scripts/tidyconfig.txt" -m "$RESPEC_SOURCE"
```

Comment:
Moved the respec script out of the /src folder because Astro doesn't actually like these files in there.
Running the script still works when I checked locally, but verification is welcome.